### PR TITLE
Release 3.1.1

### PR DESCRIPTION
**Pull request by Technoboy-**
### Motivation
Add doc for release 3.1.1

- [x] `doc` 
- [ ] `doc-required` 
- [ ] `doc-not-needed` 
- [ ] `doc-complete`

### CODE REVIEW

File: data/release-pulsar.js [hunk 1]
```diff
@@ -1,4 +1,13 @@
 module.exports = [
+    {
+      "author": "Technoboy-",
+      "tagName": "v3.1.1",
+      "publishedAt": "2023-10-24T16:37:40Z",
+      "vtag": "3.1.x",
+      "releaseNotes": "/release-notes/versioned/pulsar-3.1.1/",
+      "doc": "/docs/3.1.x",
+      "version": "v3.1.x"
```
Comment (tisonkun) [inline data/release-pulsar.js:9]: Remove `version` content in L18.

File: data/release-java.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 module.exports = [
+{tagName: "v3.1.1",vtag:"3.1.x",releaseNotes:"/release-notes/versioned/client-java-3.1.1/",doc:"/docs/3.1.x/client-libraries-java",version:"v3.1.x"},
 {tagName: "v3.1.0",vtag:"3.1.x",releaseNotes:"/release-notes/versioned/client-java-3.1.0/",doc:"/docs/3.1.x/client-libraries-java",version:"v3.1.x"},
```
Comment (tisonkun) [inline data/release-java.js:3]:
```suggestion
{tagName: "v3.1.0",vtag:"3.1.x",releaseNotes:"/release-notes/versioned/client-java-3.1.0/",doc:"/docs/3.1.x/client-libraries-java",version:""},
```

This is somehow cumbersome..But we don't make a script for automatically doing so yet.

File: data/release-pulsar.js (hunk 1 above)
Comment (Technoboy-) [reply data/release-pulsar.js]: thanks, fixed

File: data/release-java.js (hunk 2 above)
Comment (Technoboy-) [reply data/release-java.js:3]: fixed

**Review by tisonkun — APPROVED** [review]
Thank you!

Cross refer to the VOTE RESULT thread - https://lists.apache.org/thread/n030qsklr9bb9dqv48lddztl172hnrs9

**Pull request merged**